lib/glimesh/accounts.ex

```diff
@@ -88,6 +88,11 @@ defmodule Glimesh.Accounts do
   """
   def get_user!(id), do: Repo.get!(User, id)
 
+  def is_user_banned_by_username(username) do
```

Comment:
```suggestion
  def is_user_banned_by_username?(username) do
```
I really like the convention of using trailing question marks on boolean functions, what do you think?

Reply:
I think that would be a good convention, quick and easy way of seeing what the function is going to return.